node-api/src/events/mod.rs

```diff
@@ -171,25 +177,7 @@ mod tests {
 
 	/// Compare some actual [`RawEventDetails`] with a hand-constructed
 	/// (probably) [`TestRawEventDetails`].
-	pub fn assert_raw_events_match(
-		// Just for convenience, pass in the metadata type constructed
-		// by the `metadata` function above to simplify caller code.
-		metadata: &Metadata,
-		actual: EventDetails<H256>,
-		expected: TestRawEventDetails,
-	) {
-		let types = &metadata.runtime_metadata().types;
-
-		// Make sure that the bytes handed back line up with the fields handed back;
-		// encode the fields back into bytes and they should be equal.
-		let actual_fields = actual.field_values().expect("can decode field values (1)");
-		let mut actual_bytes = vec![];
-		for field in actual_fields.into_values() {
-			crate::scale_value::encode_as_type(&field, field.context, types, &mut actual_bytes)
-				.expect("should be able to encode properly");
-		}
-		assert_eq!(actual_bytes, actual.field_bytes());
```

Comment:
This assert failed after upgrading the scale libraries and was removed in subxt as well. I simply followed suit, didn't look  further into it. Since all the other tests are passing, I'm inclined to believe this is alright. Either way, there is a follow up issue: https://github.com/scs/substrate-api-client/issues/688